util/warnp.c

```diff
@@ -17,6 +17,10 @@ static void
 done(void)
```

Comment:
The comment on this function needs updating.

Reply:
done